packages/vuetify/src/components/VAlert/VAlert.tsx

```diff
@@ -138,6 +141,29 @@ export const VAlert = genericComponent<VAlertSlots>()({
       },
     }))
 
+    // Helper function to wrap slot content with defaults
+    function wrapSlot (slotName: string, slotFn: (() => any) | undefined, fallbackContent?: any) {
+      const slotDefaultsInfo = getSlotDefaultsInfo(slotName)
+
+      if (!slotDefaultsInfo && !slotFn) {
+        return fallbackContent
+      }
+
+      if (!slotDefaultsInfo) {
+        return slotFn?.() ?? fallbackContent
+      }
+
+      const { componentDefaults, directProps } = slotDefaultsInfo
+
+      return (
+        <VDefaultsProvider defaults={ componentDefaults as any }>
+          <div { ...directProps }>
```

Comment:
Not every slot should create its own element, this probably breaks a lot of styles. 

```
'#text': {
  class: 'pl-4 py-4',
},
```

should either only work only for specific slots or not be supported at all.